os/mm/kmm_heap/kmm_calloc.c

```diff
@@ -115,9 +117,17 @@ void *kmm_calloc_at(int heap_index, size_t n, size_t elem_size)
 	kheap = kmm_get_heap();
 #ifdef CONFIG_DEBUG_MM_HEAPINFO
 	ARCH_GET_RET_ADDRESS
-	return mm_calloc(&kheap[heap_index], n, elem_size, retaddr);
+	ret = mm_calloc(&kheap[heap_index], n, elem_size, retaddr);
+	if (ret == NULL) {
```

Comment:
how about adding mdbg message here, to identify from which function allocation failing

Reply:
Added. Thanks.